src/main/java/org/terasology/books/rendering/nui/layers/BookScreen.java

```diff
@@ -65,56 +73,50 @@
     private static final String STATUS_READING = "Reading";
     private static final String STATUS_READ_ONLY = "Read-only";
 
-    private final Logger logger = LoggerFactory.getLogger(BookScreen.class);
-
-    @In
-    private NUIManager nuiManager;
-    @In
-    private LocalPlayer localPlayer;
-    @In
-    private ClipboardManager clipboardManager;
+    private static final Logger logger = LoggerFactory.getLogger(BookScreen.class);
+    static List<String> pages;
+    static boolean leftPageEditing = true;
+    static Binding<Integer> index;
     @In
-    private PrefabManager prefabManager;
+    private static PrefabManager prefabManager;
```

Comment:
@skaldarnar Yes, the since only a single BookScreen instance can stay active at any point of time, the usage of static everywhere in this class is intended. It is primarily done so to make some variables and methods accessible to the PageEditor screen which is another such class in this module.